src/scripts/main.js

```diff
@@ -222,3 +215,36 @@ chrome.runtime.onMessage.addListener(function(request) {
 
 });
 
+const watcher = new MutationObserver(() => populateSignatures());
+
+watcher.observe(document.body, {
+	childList: true
+});
+
+// Check for and create default values 
+
+getPopupInfo()
+	.then(data => {
+		console.log(data);
+		if(data === null){
+
+			chrome.runtime.sendMessage({
+				method: 'saveFormData',
+				data: {
+					'amount': '3',
+					'enabled': 'true',
+					'include-author': 'true',
+					'include-description': 'true',
+					'include-heading': 'true',
+					'include-humanDate': 'true',
+					'include-link': 'true',
+					'include-title': 'true',
+					'rss': 'http://www.ft.com/rss/home/uk',
```

Comment:
It would need to be a part of our build process and right now, our pipeline has no way of setting a global variable that can be inserted into our scripts at build time. If you can see it causing trouble in the future, we can remove it the URL from popup.html and rely on main.js to populate it on first use with the default data.